cmd/brimcap/migrate/command.go

```diff
@@ -257,10 +256,23 @@ func (m *migration) migrateData(ctx context.Context) error {
 		return err
 	}
 	defer f.Close()
-	if _, err = m.conn.LogPostReaders(ctx, m.engine, m.poolID, nil, f); err != nil {
+	resp, err := m.conn.Add(ctx, m.poolID, f)
+	if err != nil {
 		return err
 	}
-	return nil
+	defer resp.Body.Close()
+	rec, err := zngio.NewReader(resp.Body, zson.NewContext()).Read()
+	if err != nil {
+		return err
+	}
+	var add api.AddResponse
+	if err := zson.UnmarshalZNGRecord(rec, &add); err != nil {
+		return err
+	}
+	return m.conn.Commit(ctx, m.poolID, add.Commit, api.CommitRequest{
+		Author:  "brimcap",
+		Message: "automatic migration of " + zngpath,
+	})
```

Comment:
Gosh, I f'in hate that this is here.

Reply:
Should our next step be to move the `api.AddResponse` unmarshaling into `api/client.Connection.Add`?

Reply:
Possibly, but I was more lamenting that brimcap still has the capability to post to a lake service. I should have written this stuff in the brim library in retrospect.